treatment-service/server/server.go

```diff
@@ -98,6 +100,11 @@ func NewServer(configFiles []string) (*Server, error) {
 		mux.Handle("/schema.yaml", web.FileHandler(path.Join(cfg.SwaggerConfig.OpenAPISpecsPath, "schema.yaml"), false))
 	}
 
+	subscribe := false
+	if cfg.MessageQueueConfig.Kind != config.NoopMQ {
+		subscribe = true
+	}
```

Comment:
More like a nit than anything, but can we not use the `Server`'s `appContext` to determine whether the `MessageQueueService` is of the `NoopMQ` type or not? 😅 But I guess the type checking of that would make things more bulky and long-winded too :/

Reply:
Hmm not sure what you mean, I'm using the config here right? 😮

Reply:
Ah I was just wondering if there was a way to avoid creating a new `subscribe` field in the `Server` struct and instead perform checks on the `appContext`'s `MessageQueueService` to see if it needs to subscribe to a message queue topic. But it's just a tiny comment, no worries about it.